src/wallet/Wif.ts

```diff
@@ -1,5 +1,5 @@
 // Stable
-import { instantiateSecp256k1, instantiateSha256 } from "@bitauth/libauth";
+import {instantiateSecp256k1, instantiateSha256} from "@bitauth/libauth";
```

Comment:
I think your linter is configured differently from the project `prettier` configuration.

Perhaps running `yarn format` could set this back.  Otherwise it will automatically be fixed when the CI format job is run.